connect-button/src/main/java/com/ifttt/connect/ui/BaseConnectButton.java

```diff
@@ -842,7 +842,7 @@ public void onAnimationEnd(Animator animation) {
         OnClickListener startAuthOnClickListener = v -> {
             AnalyticsManager.getInstance(getContext())
                     .trackUiClick(AnalyticsObject.CONNECT_BUTTON_EMAIL,
-                    AnalyticsLocation.fromConnectButton(getContext()));
```

Comment:
Do we still need this method after making these changes?

Reply:
yes, still used for one other event.